<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="operand1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="operand2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>AND</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>&lt;=&gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>=&gt;</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XOR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NOT</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="operand3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="Result">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="reza">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rezb">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
